modules/nf-core/propr/logratio/meta.yml

```diff
@@ -0,0 +1,70 @@
+name: "propr_logratio"
+description: ALR/CLR transform the data matrix
+keywords:
+  - alr
+  - clr
+  - logratio
+  - boxcox
+  - transformation
+  - propr
+
+tools:
+  - "propr":
+      description: "Logratio methods for omics data"
+      homepage: "https://github.com/tpq/propr"
+      documentation: "https://rdrr.io/cran/propr/man/propr.html"
+      tool_dev_url: "https://github.com/tpq/propr"
+      doi: "10.1038/s41598-017-16520-0"
+      licence: ["GPL-2"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information.
+        This can be used at the workflow level to pass optional parameters to the module.
+        [id: 'test', ...]
+  - count:
+      type: file
+      description: |
+        Count matrix, where rows = variables or genes, columns = samples or cells.
+        This matrix should not contain zeros. Otherwise, they will be first replaced by the minimun value.
+        You may want to handle the zeros with a different method beforehand.
+      pattern: "*.{csv,tsv}"
+  - transformation:
+      type: string
+      description: |
+        (Optional) 'clr' or 'alr'. Default: 'clr'
+  - reference:
+      type: string
+      description: |
+        (Optional) Reference gene for ALR transformation. It can be a number refering to the gene index or a string corresponding to the gene name.
+        Default = last variable
+  - alpha:
+      type: string
+      description: |
+        (Optional) Alpha parameter for Box-Cox transformation. Default = null, standard log transformation is applied.
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information.
```

Comment:
Make sure this is appropriate to the matrix input (the meta is probably not sample info in this case)